package.json

```diff
@@ -14,82 +14,26 @@
   },
   "homepage": "https://github.com/formidablelabs/victory-animation",
   "scripts": {
-    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run build-lib",
-    "preversion": "npm run check",
-    "version": "npm run clean && npm run build && git add -A dist",
-    "clean-dist": "rimraf dist",
-    "build-dist-min": "webpack --config webpack.config.js",
-    "build-dist-dev": "webpack --config webpack.config.dev.js",
-    "build-dist": "npm run clean-dist && npm run build-dist-min && npm run build-dist-dev",
-    "clean-lib": "rimraf lib",
-    "build-lib": "npm run clean-lib && babel --stage 0 src -d lib",
-    "clean": "npm run clean-lib && npm run clean-dist",
-    "build": "npm run build-lib && npm run build-dist",
-    "server-dev": "webpack-dev-server --port 3000 --config demo/webpack.config.dev.js --content-base demo",
-    "server-hot": "webpack-dev-server --port 3000 --config demo/webpack.config.hot.js --hot --content-base demo",
-    "server-test": "webpack-dev-server --port 3001 --config webpack.config.test.js",
-    "dev": "npm run server-dev & npm run server-test",
-    "hot": "npm run server-hot & npm run server-test",
-    "open-demo": "opener http://127.0.0.1:3000",
-    "open-dev": "npm run dev & npm run open-demo",
-    "open-hot": "npm run hot & npm run open-demo",
-    "lint-node": "eslint -c .eslintrc-node *.js demo/webpack.*.js",
-    "lint-react": "eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx",
-    "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client",
-    "lint": "npm run lint-node && npm run lint-react && npm run lint-react-test",
-    "test-frontend": "node node_modules/karma/bin/karma start karma.conf.js",
-    "test-frontend-ci": "node node_modules/karma/bin/karma start --browsers PhantomJS,Firefox karma.conf.coverage.js",
-    "test-frontend-cov": "node node_modules/karma/bin/karma start karma.conf.coverage.js",
-    "test-frontend-dev": "node node_modules/karma/bin/karma start karma.conf.dev.js",
-    "test": "npm run test-frontend && echo 'TODO Server 13, Integration 12 tests.'",
-    "test-ci": "npm run test-frontend-ci",
-    "test-cov": "npm run test-frontend-cov",
-    "test-dev": "npm run test-frontend-dev",
-    "check": "npm run lint && npm run test",
-    "check-ci": "npm run lint && npm run test-ci",
-    "check-cov": "npm run lint && npm run test-cov",
-    "check-dev": "npm run lint && npm run test-dev"
+    "postinstall": "builder run npm:postinstall",
+    "preversion": "builder run npm:preversion",
+    "version": "builder run npm:version",
+    "test": "builder run npm:test"
   },
   "dependencies": {
-    "babel": "^5.5.8",
-    "babel-core": "^5.5.8",
-    "babel-loader": "^5.3.2",
-    "d3": "^3.5.6",
-    "radium": "^0.14.1",
-    "rimraf": "^2.4.0",
-    "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "webpack": "^1.10.0"
+    "builder": "^2.1.0",
```

Comment:
Open question -- we're not following semver for `v2`. Do we want to pin versions as `builder` evolves? Or just "go for it" and roll with the upgrade punches?